src/components/Preloader/preloader.module.scss

```diff
@@ -1,90 +1,66 @@
 .preloader {
-  position: fixed;
-  left: 0;
-  top: 0;
+  position: absolute;
+  top: 50%;
+  left: 50%;
+  transform: translate(-50%, -50%);
+  padding: 0;
+  margin: 0;
+  background-color: #1F1939;
+  text-align: center;
+  height: 100vh;
+  font-family: "lato";
```

Comment:
Any specific use of font family here

Reply:
Initially I thought to make D o c 2 p e n jump instead of circles. I dropped that because it looked bad. I forgot to remove these after that😅